src/ViewHelpers/VariableViewHelper.php

```diff
@@ -57,6 +57,9 @@ public static function renderStatic(
     ) {
         $value = $renderChildrenClosure();
         $renderingContext->getVariableProvider()->add($arguments['name'], $value);
+        if ($delegateVariableProvider = $renderingContext->getViewHelperVariableContainer()->getTopmostDelegateVariableContainer(RenderViewHelper::class)) {
```

Comment:
Can you split this to two lines? At least PHPStan doesn't like variable assignment in conditions. ;-)

Reply:
I can sure do that :)